From 8558c5398d6f51507e6f5ff205526b1756377d63 Mon Sep 17 00:00:00 2001
From: Bartlomiej Zolnierkiewicz <bzolnier@gmail.com>
Date: Fri, 11 Dec 2009 12:23:13 -0800
Subject: [PATCH 17/42] Staging: rt28x0: remove __LINE__ instances

commit 460bb8df1e910a8c5d36ad363fbc1f0164cf0b85 upstream.

Remove instances of __LINE__ preprocessor directive to make
validation of resulting binary output files easier.

Signed-off-by: Bartlomiej Zolnierkiewicz <bzolnier@gmail.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>
---
 drivers/staging/rt2860/common/mlme.c      |    7 ++-----
 drivers/staging/rt2860/common/rtmp_init.c |    3 ++-
 drivers/staging/rt2860/pci_main_dev.c     |    1 -
 3 files changed, 4 insertions(+), 7 deletions(-)

diff --git a/drivers/staging/rt2860/common/mlme.c b/drivers/staging/rt2860/common/mlme.c
index 9fa853e..9c250c2 100644
--- a/drivers/staging/rt2860/common/mlme.c
+++ b/drivers/staging/rt2860/common/mlme.c
@@ -1222,13 +1222,11 @@ if (OPSTATUS_TEST_FLAG(pAd, fOP_STATUS_PCIE_DEVICE))
 			{
 			if (pAd->StaCfg.PSControl.field.EnableNewPS == TRUE)
 	{
-				DBGPRINT(RT_DEBUG_TRACE, ("%s::%d\n",__FUNCTION__,__LINE__));
-
+				DBGPRINT(RT_DEBUG_TRACE, ("%s\n", __func__));
 				RT28xxPciAsicRadioOff(pAd, GUI_IDLE_POWER_SAVE, 0);
 				}
 			else
 				{
-				DBGPRINT(RT_DEBUG_TRACE, ("%s::%d\n",__FUNCTION__,__LINE__));
 				AsicSendCommandToMcu(pAd, 0x30, PowerSafeCID, 0xff, 0x2);
 				// Wait command success
 				AsicCheckCommanOk(pAd, PowerSafeCID);
@@ -1240,12 +1238,11 @@ if (OPSTATUS_TEST_FLAG(pAd, fOP_STATUS_PCIE_DEVICE))
                 {
 			if (pAd->StaCfg.PSControl.field.EnableNewPS == TRUE)
 				{
-				DBGPRINT(RT_DEBUG_TRACE, ("%s::%d\n",__FUNCTION__,__LINE__));
+				DBGPRINT(RT_DEBUG_TRACE, ("%s\n", __func__));
 				RT28xxPciAsicRadioOff(pAd, GUI_IDLE_POWER_SAVE, 0);
 				 }
 				else
 				{
-				DBGPRINT(RT_DEBUG_TRACE, ("%s::%d\n",__FUNCTION__,__LINE__));
 				AsicSendCommandToMcu(pAd, 0x30, PowerSafeCID, 0xff, 0x02);
 				// Wait command success
 				AsicCheckCommanOk(pAd, PowerSafeCID);
diff --git a/drivers/staging/rt2860/common/rtmp_init.c b/drivers/staging/rt2860/common/rtmp_init.c
index 5deba8d..1cc6e44 100644
--- a/drivers/staging/rt2860/common/rtmp_init.c
+++ b/drivers/staging/rt2860/common/rtmp_init.c
@@ -1261,7 +1261,8 @@ VOID	NICInitAsicFromEEPROM(
 			&& (pAd->StaCfg.PSControl.field.rt30xxPowerMode == 3)
 			&& (pAd->StaCfg.PSControl.field.EnableNewPS == TRUE))
 		{
-			DBGPRINT(RT_DEBUG_TRACE,("%s::%d,release Mcu Lock\n",__FUNCTION__,__LINE__));
+			DBGPRINT(RT_DEBUG_TRACE,
+				("%s, release Mcu Lock\n", __func__));
 			RTMP_SEM_LOCK(&pAd->McuCmdLock);
 			pAd->brt30xxBanMcuCmd = FALSE;
 			RTMP_SEM_UNLOCK(&pAd->McuCmdLock);
diff --git a/drivers/staging/rt2860/pci_main_dev.c b/drivers/staging/rt2860/pci_main_dev.c
index 38bc429..2c51d9d 100644
--- a/drivers/staging/rt2860/pci_main_dev.c
+++ b/drivers/staging/rt2860/pci_main_dev.c
@@ -599,7 +599,6 @@ VOID RTMPInitPCIeLinkCtrlValue(
 				}
 
 				DBGPRINT(RT_DEBUG_TRACE, ("====> rt28xx Write 0x83 Command = 0x%x.\n", PCIePowerSaveLevel));
-					       printk("\n\n\n%s:%d\n",__FUNCTION__,__LINE__);
 
 				AsicSendCommandToMcu(pAd, 0x83, 0xff, (UCHAR)PCIePowerSaveLevel, 0x00);
 			}
-- 
1.7.1

